trace-obfuscation/Cargo.toml

```diff
@@ -14,6 +14,7 @@ serde = { version = "1.0.145", features = ["derive"] }
 serde_json = "1.0"
 url = "2.4.0"
 percent-encoding = "2.1"
+lazy_static = "1.4"
```

Comment:
I would use OnceCell variants. It's part of the std so less dependencies and provides some advantages over lazy_static since it can accept closures as initializers, it's macro-free (I wouldn't count that as a massive adv) and, on certain scenarios, it could be more efficient regarding memory handling.
```suggestion
```